tests/test_cli.py

```
@@ -2360,3 +2360,74 @@ def test_runtime_precondition_trap_non_json(
+        result = subprocess.run(
+            [sys.executable, "-m", "vera.cli", "run", "/dev/stdin"],
+            input=self.SIMPLE_PROGRAM,
+            capture_output=True,
+            text=True,
+        )
+        assert result.returncode == 0, result.stderr
+        assert "42" in result.stdout
+
+    def test_verify_dev_stdin_subprocess(self) -> None:
+        """vera verify /dev/stdin works end-to-end."""
+        result = subprocess.run(
```

Ruff (0.15.7)

[error] `subprocess` call: check for execution of untrusted input

(S603)